minigalaxy/ui/gametile.py

```diff
@@ -566,3 +566,561 @@ def update_to_state(self, state):
         self.current_state = state
         if state in self.STATE_UPDATE_HANDLERS:
             self.STATE_UPDATE_HANDLERS[state](self)
+
+
+@Gtk.Template.from_file(os.path.join(UI_DIR, "gametilelist.ui"))
+class GameTileList(Gtk.Box):
```

Comment:
Could this be put in its own file or would that cause issues?

Reply:
Loos like it works, don't see any bugs.